package.json

```diff
@@ -59,8 +59,8 @@
   },
   "scripts": {
     "pre-test": "cross-env NODE_ENV=test npm run undo:db && cross-env NODE_ENV=test npm run migrate:db && cross-env NODE_ENV=test npm run seed:db",
-    "test": "npm run pre-test && cross-env NODE_ENV=test nyc --reporter=text --reporter=html mocha --timeout=5000 --require @babel/polyfill --require @babel/register --exit ./src/test/index.test.js",
-    "stop-linux": "killall node",
+    "test": "npm run pre-test && cross-env NODE_ENV=test nyc --reporter=text --reporter=html mocha --timeout=50000 --require @babel/polyfill --require @babel/register --exit ./src/test/index.test.js",
```

Comment:
Don't you think 50 seconds is too much? 🤔

Reply:
Yes, its actually alot, i reduced it to 10000

Reply:
we are still getting a timeout problem. I have moved it up to 20000